### PR TITLE
Version change to 0.1.0

### PR DESCRIPTION
**Pull request by maqi**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/maidsafe_vault/112)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @chandraprakash

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,3 +15,35 @@
...
+    - local joining test (process counting)
+    - network example (nodes populating)
+- SafeCoin farming (new persona may need to be introduced, the task needs to be ‘expandable’ ) documentation
+    - farming
+    - account
+- Implement handling for Safecoin farming rate
+    - Farming rate determined by the Sacrificial copies.  
+    - Farming rate drops when more copies are available and rises when less copies are available.
+
+
```
Comment (chandraprakash) [inline CHANGELOG.md]: You need to retain the task links (JIRA/Issues)

**Comment by Viv-Rajkumar** [comment]
Review status: 0 of 3 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

<sup>**[Cargo.toml, line 4 \[r2\]](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112#-JsQe--RI_i5CfnYitzU)** ([raw file](https://github.com/maidsafe/maidsafe_vault/blob/ee6bcc715ca88a5064150246908b6cfb0a4dec74/Cargo.toml#L4)):</sup>
This PR should be a minor version bump, not patch version. so it should be going to `0.1.0`

---

<sup>**[CHANGELOG.md, line 49 \[r2\]](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112#-JsPuilqC3XUngl_U6_6-r2-49)** ([raw file](https://github.com/maidsafe/maidsafe_vault/blob/ee6bcc715ca88a5064150246908b6cfb0a4dec74/CHANGELOG.md#L49)):</sup>
also please keep the change log in descending order.

---

<sup>**[README.md, line 29 \[r2\]](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112#-JsQeX12H-jspeOyM-R0)** ([raw file](https://github.com/maidsafe/maidsafe_vault/blob/ee6bcc715ca88a5064150246908b6cfb0a4dec74/README.md#L29)):</sup>
`0.0.5` version needs to be updated accordingly based on minor version change. Also please remove `code clean up`. Tasks need to be specific or can be added here with a corresponding JIRA task that which is covering the details.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112)

**Comment by maqi** [comment]
I am planning to create some jira tasks regarding the code cleanup work,
which I think shall be a part of next sprint.
That's why I think it shall stay in the README.md

**Comment by maqi** [comment]
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

<sup>**[Cargo.toml, line 4 \[r2\]](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112#-JsQe--RI_i5CfnYitzU)** ([raw file](https://github.com/maidsafe/maidsafe_vault/blob/ee6bcc715ca88a5064150246908b6cfb0a4dec74/Cargo.toml#L4)):</sup>
Done.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112)

**Comment by maqi** [comment]
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

<sup>**[CHANGELOG.md, line 49 \[r2\]](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112#-JsPuilqC3XUngl_U6_6-r2-49)** ([raw file](https://github.com/maidsafe/maidsafe_vault/blob/ee6bcc715ca88a5064150246908b6cfb0a4dec74/CHANGELOG.md#L49)):</sup>
Done.

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112)

**Comment by maqi** [comment]
Review status: 0 of 4 files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

<sup>**[README.md, line 29 \[r2\]](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112#-JsQeX12H-jspeOyM-R0)** ([raw file](https://github.com/maidsafe/maidsafe_vault/blob/ee6bcc715ca88a5064150246908b6cfb0a4dec74/README.md#L29)):</sup>
I am planning to create some jira tasks regarding the code cleanup work,
which I think shall be a part of next sprint.
That's why I think it shall stay in the README.md

---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/maidsafe/maidsafe_vault/112)

**Pull request closed without merging**